extensions/wikia/Oasis/Oasis.i18n.php

```diff
@@ -2166,28 +2157,28 @@
 	'oasis-toolbar-not-enabled-here' => 'このツールはこのWikiaではご利用いただけません。',
 	'oasis-comments-header' => '$1件の{{PLURAL:$1|コメント|コメント}}',
 	'oasis-comments-showing-most-recent' => '最近の$1件を表示しています',
-	'oasis-comments-anonymous-prompt' => '匿名ユーザー<br /> [[Special:UserLogin|ログインしますか？]]',
+	'oasis-comments-anonymous-prompt' => '匿名ユーザー<br /> <a href="#" class="ajaxLogin">ログインしますか？</a>',
```

Comment:
This will break because this message is passed to parser - version in app repo is correct here. AjaxLogin extension is long gone anyways :)